docs/open-source-tizen/tizen-studio/docs/example-web-certi-ext.md

```diff
@@ -0,0 +1,91 @@
+# Modifying the text in Certificate Manager
+
+## Step 1
+
+#### Identifier
+- org.tizen.common.sign.ui.certificateTypes
+
+## Step 2
+
+#### Description
+- This extension point is used for registering extension certificate type.
+
+## Step 3
+
+#### Configuration
+```text
```

Comment:
This code block is xml, but text is okay if you want to focus on description of xml elements.